style.css

```diff
@@ -684,6 +691,124 @@ label {
   .active {
     display: none;
   }
+
+  #modal-overlay {
+    backdrop-filter: blur(30px);
+    filter: brightness(80%);
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 9999999;
+  }
```

Comment:
- [ ] Also, please consider taking a look at the mobile styles for the popup, at the moment this is what I'm getting 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/233494831-038ced56-7b65-432e-add5-40df79809581.png) | ![image](https://user-images.githubusercontent.com/77069011/233494681-47a5ee95-f0ed-4e84-b1ff-3a93e6619dbc.png) |

- [ ] You could also improve a little the desktop version by adding the icons to the buttons and using the correct spacing and font-sizes! 🤓 🔧 

|Template|Your design|
|---|---|
| ![image](https://user-images.githubusercontent.com/77069011/233495197-d5c6c70c-d17e-4fbf-a526-a22d3ee40b5c.png) | ![image](https://user-images.githubusercontent.com/77069011/233495052-30441393-8a23-4303-bca7-14b7cf4559de.png) |

Please consider fixing these! 🤖